src/components/common/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import { deleteToken, getToken } from 'utils/localStorage';
+
+import BalckButton from './BalckButton';
+
+const Header = () => {
+  const [isSignedIn, setIsSignedIn] = useState(!!getToken());
+  const handleLogOut = () => {
+    setIsSignedIn(false);
+    deleteToken();
+  };
+
+  return (
+    <div className="fixed bg-white/70 px-10 flex justify-between items-center w-full h-14 left-0">
+      <Link className="text-black font-bold  text-2xl flex items-center  gap-1" to="/">
+        Todo Master
+      </Link>
+      <div className="flex gap-10">
+        <NavLink className="text-slate-500 relative flex items-center" to="/">
+          Home
+        </NavLink>
+        <NavLink className="text-slate-500 relative  flex items-center" to="/todo">
+          Todo List
+        </NavLink>
+      </div>
+      <p />
+
+      {isSignedIn && isSignedIn ? (
```

Comment:
같은 변수를 두번 확인할 필요는 없을 것 같습니다

```suggestion
      {isSignedIn ? (
```